HopsanGUI/Widgets/LibraryWidget.cpp

```diff
@@ -696,7 +720,13 @@ void LibraryWidget::handleItemDoubleClick(QTreeWidgetItem *item, int column)
                 basePath.append("/");
             }
             QString sourceFile = appearance->getSourceCodeFile();
-            gpModelHandler->loadTextFile(basePath+sourceFile);
+            if(!sourceFile.isEmpty()) {
```

Comment:
I am not able to trigger these double click events on any library tree items. (I dont think they are necessary so no big deal)

Reply:
They only work for external libraries. I can change it so that they also work for default components. It makes no sense why it should not.